js/adapt-contrib-slider.js

```diff
@@ -422,7 +394,8 @@ define(function(require) {
             if (correctAnswer) {
                 // Check that correctAnswer is neither undefined nor empty
                 answers.push(correctAnswer);
-            } else if (bottom !== undefined) {
+            } else if (bottom && top) {
```

Comment:
This if statement probably won't work as expected if `_bottom` has been set to `0`.